patches/0006-docs-patching.patch

```diff
@@ -664,38 +753,36 @@ index a2f6f69ba..2f7bc1ff3 100644
    a Stackdriver Monitoring TimeSeries.list API call.
    This filter is used to select a specific TimeSeries for
 diff --git a/website/docs/r/compute_backend_bucket_signed_url_key.html.markdown b/website/docs/r/compute_backend_bucket_signed_url_key.html.markdown
-index 9bce1ba3a..f937568e0 100644
+index 9bce1ba3a..6bac2b55a 100644
 --- a/website/docs/r/compute_backend_bucket_signed_url_key.html.markdown
 +++ b/website/docs/r/compute_backend_bucket_signed_url_key.html.markdown
-@@ -28,9 +28,8 @@ To get more information about BackendBucketSignedUrlKey, see:
+@@ -28,10 +28,6 @@ To get more information about BackendBucketSignedUrlKey, see:
  * How-to Guides
      * [Using Signed URLs](https://cloud.google.com/cdn/docs/using-signed-urls/)
  
 -~> **Warning:** All arguments including the following potentially sensitive
 -values will be stored in the raw state as plain text: `key_value`.
 -[Read more about sensitive data in state](https://www.terraform.io/language/state/sensitive-data).
-+~> **Warning:** All arguments including `key_value` will be stored in the raw
-+state as plain-text.
- 
+-
  ## Example Usage - Backend Bucket Signed Url Key
  
+ 
 diff --git a/website/docs/r/compute_backend_service.html.markdown b/website/docs/r/compute_backend_service.html.markdown
-index 07e776730..025685511 100644
+index 07e776730..3a6c88b9c 100644
 --- a/website/docs/r/compute_backend_service.html.markdown
 +++ b/website/docs/r/compute_backend_service.html.markdown
-@@ -34,9 +34,8 @@ To get more information about BackendService, see:
+@@ -34,10 +34,6 @@ To get more information about BackendService, see:
  * How-to Guides
      * [Official Documentation](https://cloud.google.com/compute/docs/load-balancing/http/backend-service)
  
 -~> **Warning:** All arguments including the following potentially sensitive
 -values will be stored in the raw state as plain text: `iap.oauth2_client_secret`, `iap.oauth2_client_secret_sha256`.
 -[Read more about sensitive data in state](https://www.terraform.io/language/state/sensitive-data).
-+~> **Warning:** All arguments including `iap.oauth2_client_secret` and `iap.oauth2_client_secret_sha256` will be stored in the raw
```

Comment:
Are these secrets? If so, isn't this still wrong?

Reply:
> isn't this still wrong?

What do you mean?

Reply:
Ah, I see, the note is still here... let me check why!

Reply:
There's a few more notes like that, not just this one.
For example, `website/docs/r/iap_client.html.markdown`

Reply:
This looks like a github rendering bug to me. There is no green text in the actual patch file.

Reply:
Yeah, looks like the diff is buggy - when I viewed the whole file that text disappears.

Reply:
You can see the note is gone e.g. in sdk/dotnet/Iap/Client.cs below